src/ray/protobuf/core_worker.proto

```diff
@@ -285,6 +285,21 @@ message PlasmaObjectReadyRequest {
 message PlasmaObjectReadyReply {
 }
 
+message SpillObjectsRequest {
```

Comment:
Can you document these calls better (i.e., who is calling and for what reasons?)

Reply:
sure